notebooks/cookbook/Querying and Modifying Charges and Other Parameters.ipynb

```diff
@@ -98,12 +98,81 @@
     "    if particle1 == 0 or particle2 == 0:\n",
     "        print(f'Particles ({particle1}, {particle2}), length = {length}, k = {k}')"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "50f5157c",
+   "metadata": {},
+   "source": [
+    "In some cases you may want to modify those parameters to make a bond behave differently from what the `ForceField` assigned.\n",
+    "We show you here that to modify these parameters are also easy."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cd5396c5",
+   "metadata": {},
+   "source": [
+    "Let's say we would want to modify the force constant of the bond connecting particle 1 and 0 to a new number `_k`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "0dc0db66",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "k_ = 2666 * unit.kilojoule / ( unit.nanometer ** 2 * unit.mole )"
```

Comment:
This can be written more simply as `unit.kilojoules_per_mole/unit.nanometer**2`.  Also, since this value only ever appears once, we can just put it directly in the call to `setBondedParameters()` instead of defining a separate variable for it.